tests/data/gamma_data.py

```diff
@@ -462,3 +462,11 @@ def geodesic_shape_test_data(self):
             ),
         ]
         return self.generate_tests([], random_data)
+
+    def scalar_curvature_test_data(self):
+        random_data = [
+            dict(point=self.Space().random_point(), atol=gs.atol),
+            dict(point=self.Space().random_point(2), atol=gs.atol),
+            dict(point=self.Space().random_point(3), atol=gs.atol),
```

Comment:
you can remove `atol=gs.atol`: it will be the one used by default.

Reply:
thanks, done